shift_detector/detector.py

```diff
@@ -60,9 +64,13 @@ def run(self, *checks, logger_level=logger.ERROR):
             class_names = map(lambda c: c.__class__.__name__, checks)
             raise Exception("All elements in checks should be a Check. Received: {}".format(', '.join(class_names)))
 
+        download_nltk = False
```

Comment:
In my opinion the download of nltk should not be placed in the Detector. Because the detector will not always be called to execute these checks. Instead it should be placed in the checks / precalculation that need nltk.